clients/src/main/java/org/apache/kafka/clients/admin/AdminClientConfig.java

```diff
@@ -149,6 +155,16 @@
                                         atLeast(0),
                                         Importance.MEDIUM,
                                         REQUEST_TIMEOUT_MS_DOC)
+                                .define(SOCKET_CONNECTIONS_SETUP_TIMEOUT_MS_CONFIG,
+                                        Type.LONG,
+                                        10 * 1000,
+                                        Importance.MEDIUM,
+                                        CommonClientConfigs.SOCKET_CONNECTIONS_SETUP_TIMEOUT_MS_DOC)
+                                .define(SOCKET_CONNECTIONS_SETUP_TIMEOUT_MAX_MS_CONFIG,
+                                        Type.LONG,
+                                        127 * 1000,
```

Comment:
Can we define the defaults in `CommonClientConfigs`?

Reply:
I think the current practice is to hard code the defaults in each client's config definition. Do we still want to define the defaults in CommonClientConfigs?

Reply:
While I also recognize that we are not consistent with this, I would do it as suggested by Rajini. The defaults are the same everywhere so it makes sense to have it defined centrally for now.